pkgs/development/python-modules/tokenizers/default.nix

```diff
@@ -0,0 +1,123 @@
+{ stdenv
+, rustPlatform
+, fetchFromGitHub
+, fetchurl
+, maturin
+, pip
+, pytest
+, python
+, pythonRecompileBytecodeHook
```

Comment:
IIRC, the hook is there for all buildPython{Application,Packages}, it's just default to `dontUsePythonRecompileBytecode = true;` due to too many package regressions.

In fact, there's no way to reverse it currently https://github.com/NixOS/nixpkgs/blob/42ea899130764441776dc6932eaa1de165eae873/pkgs/development/interpreters/python/mk-python-derivation.nix#L169
```suggestion
```

Reply:
Note that this does not use `buildPythonPackage`, so we need to add the hook explicitly.